### ISSUE TITLE
Getting current location when the app in minimized or screen in locked

### ISSUE THREAD

**Issue by W2YAdmin** [Baseflow/flutter-geolocator#372]
## 🐛 Bug Report

I am working on an app which requires me to track the user location consistently still the app is open. While it works great till the screen is on as soon as the screen is turned off (Phone locked) the getLocation starts returning the same location which I had before turning the screen off.

As I am plotting these locations on a graph, this results in a same point till the screen turns on and when it does the new location is delivered creating a long straight line.

### Expected behaviour
Get the location even when the screen of off

### Reproduction steps
Create a simple app and print current location every second
Check the debug statements when screen if off

### Configuration

**Version:** 
geolocator: ^5.1.5

**Platform:** 
- [x] :iphone: iOS
- [X] :robot: Android

**Comment by luisestudioalfa** [comment]
Hi,
I have the same problem. The moment the app goes background or when the phone is locked, the position returned is the same. I have to open the app so the geolocation is changed to the new position.

Did you manage to solve this?

**Comment by ibrierley** [comment]
Just wondering, maybe you need a wakelock for it to work, or something to enable background location to work.

**Comment by luisestudioalfa** [comment]
Yep, in the end i used the workmanager to wake the device and make a call to get the position.

**Comment by hexdump2002** [comment]
> Yep, in the end i used the workmanager to wake the device and make a call to get the position.

@luisestudioalfa may we have a look at your solution? I mean, could you post code or explain what exactly you did?

**Comment by mvanbeusekom** [comment]
Fixed in version 6.0.0 (although acquiring location updates in the background is not support, this is tracked in #53)

**Comment by hordeariibarley** [comment]
Sorry for posting it here again, but if the screen is locked, Geolocator should still work on 6.0.0^? or is this background support?

I am adding the position on a list like the code below, but if the screen is locked, no position is added. I am using a Samsung to test (model SM-A107M and Android 10) with "always" GPS config. Am I losing something?

```_positionStreamSubscription = GeolocatorPlatform.instance
        .getPositionStream(desiredAccuracy: LocationAccuracy.best)
        .listen((Position position) {
      setState(() {
        _positions.add(position);
      });
}),```

I started learning to code this year, sorry if it is a dummy question.
Thanks for the amazing package
